en/docs/learn/design-api/advanced-topics/control-api-visibility-and-subscription-availability-in-developer-portal.md

```diff
@@ -123,6 +123,48 @@ When using the REST API directly, the subscription availability options are avai
     </tr>
     </tbody>
     </table>
+    
+    
+## Control API Document Visibility in Developer Portal
+
+By default, any document associated with an API has the same visibility level of the API. That is, if the API is public, 
+its documentation is also visible to all users (registered and anonymous).
+
+!!! note
+    This feature has been disabled in the default setup.
+     
+To enable other visibility levels to the documentation, go to the  
+`<API-M_HOME>/repository/conf/deployment.toml`, 
+add the following configuration.
+
+``` toml
+[apim.publisher]
+enable_api_doc_visibility = "true"
+```
+
+Document visibility levels can be one of the following options:
+
+- **Same as API visibility:** visible to the same user roles who can see the API. For example, if the API's visibility is public, its documentation is visible to all users.
+- **Owner Only:** visible to all registered users in the API's tenant domain.
+- **Private:** visible only to the users who have permission to log in to the API Publisher web interface and create and/or publish APIs to the Developer Portal.
+
+
+### Control API document visibility using the API Publisher
+
+1.  Sign in to the API Publisher as an API creator using the following URL: 
+
+     `https://<localhost>:9443/publisher`
+
+2.  [Create a new API]({{base_path}}/learn/design-api/create-api/create-a-rest-api/) or edit an existing API.
+
+3.  In the **Documents** tab, Click **Add New Document**, to see a new drop-down list added to select visibility from. 
+Select the desired visibility level for the document.
+     
+     [ ![API document visibility selector]({{base_path}}/assets/img/learn/api-doc-visibility-selector.png) ](/{{base_path}}/assets/img/learn/api-doc-visibility-selector.png)
+     [ ![API document visibility selector dropdown]({{base_path}}/assets/img/learn/api-doc-visibility-dropdown.png) ](/{{base_path}}/assets/img/learn/api-doc-visibility-dropdown.png)
+
+4.  Add the Document.
```

Comment:
What are the visibility levels for the publisher?

Reply:
There's no visibility control in publisher. Every user who has access to Publisher portal can see all the documents.